pkgs/development/python-modules/digi-xbee/default.nix

```diff
@@ -2,21 +2,39 @@
 
 buildPythonPackage rec {
   pname = "digi-xbee";
-  version = "1.3.0";
+  version = "1.4.0";
   disabled = isPy27;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "2ed798faee0853bf7ae9ca5aa4bdcbab496e3c2d56c9f0719a8e3e0d13270891";
+    sha256 = "664737d1aab453ea40b9745f1ee1e88920acff1cce2e07c42e7f5aa64a16e6aa";
   };
 
   propagatedBuildInputs = [ pyserial srp ];
 
   # Upstream doesn't contain unit tests, only functional tests which require specific hardware
   doCheck = false;
 
+  pythonImportsCheck = [
+    "digi.xbee.models"
+    "digi.xbee.packets"
+    "digi.xbee.util"
+    "digi.xbee.comm_interface"
+    "digi.xbee.devices"
+    "digi.xbee.exception"
+    "digi.xbee.filesystem"
+    "digi.xbee.firmware"
+    "digi.xbee.io"
+    "digi.xbee.profile"
+    "digi.xbee.reader"
+    "digi.xbee.recovery"
+    "digi.xbee.sender"
+    "digi.xbee.serial"
+    "digi.xbee.xsocket"
+  ];
+
   meta = with lib; {
-    description = "Python library to interact with Digi International's XBee radio frequency modules";
+    description = "Python library to interact with Digi International's XBee radio frequencys";
```

Comment:
```suggestion
    description = "Python library to interact with Digi International's XBee radio frequencies";
```

Reply:
I wonder why I changed the description at all.